src/components/common/SearchBar.tsx

```
@@ -0,0 +1,16 @@
+import { MdSearch } from 'react-icons/md';
+
+export default function SearchBar({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)